src/docs/implementations/instana.adoc

```diff
@@ -0,0 +1,24 @@
+= Micrometer Instana
+Fabian Lange <fabian.lange@instana.com>
+:toc:
+:sectnums:
+:system: instana
+
+Instana is an automatic application performance management and infrastructure monitoring system.
+
+== Installation and Configuration
+
+Instana will automatically detect and report all metrics without the need of any additional dependency or configuration!
```

Comment:
I'm not familiar with Instana, but I assume it works as a Java agent. Doesn't this agent need to be configured for the JVM? Perhaps that is what was meant by no "additional" configuration. Maybe this is apparent to Instana users, but the current wording left me wondering how I could use Micrometer with Instana. Of course, it's fine to link to Instana's documentation that explain this so it isn't duplicated and can be kept up to date in one place.

Reply:
It does use a Java agent. But it does not need to be configured, it works automatically. I will make this clearer as well.